activiti-core/activiti-spring-boot-starter/src/main/java/org/activiti/spring/boot/ShutdownListener.java

```diff
@@ -20,7 +20,12 @@
 import org.slf4j.LoggerFactory;
 import org.springframework.context.ApplicationListener;
 import org.springframework.context.event.ContextClosedEvent;
+import org.springframework.core.Ordered;
+import org.springframework.core.annotation.Order;
+import org.springframework.stereotype.Component;
 
+@Component
```

Comment:
we are defining the bean in ProcessEngineAutoConfiguration so we should configure there.
Can you try to build an image and check the log on an env? thanks